spec/factories.rb

```diff
@@ -0,0 +1,9 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :venue do
+    address { Faker::String.random(10) }
```

Comment:
it's not really necessary, but Faker, or his forks definitely able to create fake addresses, phones and so on.

Reply:
done